app/controllers/clients_controller.rb

```diff
@@ -1,4 +1,5 @@
 class ClientsController < ApplicationController
+  include FormInputsHelper
```

Comment:
I see that you're including the `FormInputsHelper` but are you using it? For example, look at how the code is written for the income_range field: 
`<%= form.select :estimated_household_income, options_for_select(income_range, @client.estimated_household_income) %>`

It looks like they are accessing the value from the `FormInputsHelper`...is there a reason these other methods need to be in the controller?